pinot-controller/src/main/java/org/apache/pinot/controller/api/access/AccessControl.java

```diff
@@ -30,9 +30,35 @@
   /**
    * Return whether the client has data access to the given table.
    *
+   * Note: This method is only used for read access. It will be deprecated soon and its usage will be replaced by
```

Comment:
You mean, this method will be removed soon, right? In that case, you should mark it deprecated, and leave it here for one release.

Reply:
Will do.